### ISSUE TITLE
Defaults to Vaadin 7.5.9 rather than current 7.7.x for new project

### ISSUE THREAD

**Issue by basil-bourque** [vaadin/netbeans-plugin#4]
The title line says it all:
The "Vaadin Plug-in for NetBeans" creates a new project using Vaadin 7.5.9
rather than the current Vaadin 7.7.x.

Using NetBeans 8.2 with the "Vaadin Plug-in for NetBeans" version 1.1.0
(not yet certified for NetBeans 8.2, but manually installed) on macOS El
Capitan with Java 8 Update 111.

See explanation about this problem by Denis Anisimov on my Question posted to Stack Overflow:
http://stackoverflow.com/q/40274294/642706

I also filed this as ticket # 20424 on TRAC.
https://dev.vaadin.com/ticket/20424

**Comment by phansson** [comment]
I haven't been able to reproduce this, neither on NB 8.1 or NB 8.2. At the time when I write this the latest Vaadin version is v7.7.6.

Here's what I do:
I generate a new Vaadin Multi-module project like this:
![2017-01-30 21_24_00-vaadingridactionrenderertest - vaadingridactionrenderertest-widgetset - netbeans](https://cloud.githubusercontent.com/assets/1903550/22440324/46918d74-e733-11e6-8b0f-8491eeb625b6.png)

I can see in NB output window that the plugin generates the project and sub-projects using the latest Maven archetype version. Good.

Now that the project is generated I can open the POM on the "parent" project. I see the following:
![2017-01-30 21_25_47-vaadinvertest - vaadinvertest-parent - netbeans ide 8 2](https://cloud.githubusercontent.com/assets/1903550/22440344/5f82520a-e733-11e6-936a-ae939b70a2de.png)
Good!.

I can also right-click the "ui" project (the plugin doesn't recognize the "parent" project as a Vaadin project), and select "Properties". I then see:
![2017-01-30 21_28_28-vaadinvertest - vaadinvertest-ui - netbeans ide 8 2](https://cloud.githubusercontent.com/assets/1903550/22440420/b19d3370-e733-11e6-880e-f427f3b2143c.png)
Also good.

The only thing I would mention is that you shouldn't attempt to change Vaadin version from the Properties view of the "ui" project. It will generate something like this

```
   <properties>
        <vaadin.version>x.x.x</vaadin.version>
   </properties>
```

in the POM of your UI project, not the parent project. So now you have the version defined in two locations, where one overrides the other. This is confusing and probably not intended.

Bottom line: You should control the Vaadin version by manually editing the `<properties>` section in the parent POM, not anywhere else. Avoid the temptation to change version from the Properties window.

**Comment by denis-anisimov** [comment]
Yes, thank you for reporting.
It is known issue.
